### ISSUE TITLE
Export / Import System

### ISSUE THREAD

**Issue by Tangol** [nivocart/nivocart#17]
As per my report [here](https://github.com/villagedefrance/OpenCart-Overclocked/issues/152#issuecomment-372821107), the **Export / Import XLSX and CSV** system doesn't seem to work.

How to test:
- Install latest version of NC with demo data
- Export XLSX - Products Options (or any other)
- Delete all Products Options
- Refresh the admin page
- Import XLSX with "_No (Delete all old data before Import)_"
- Result: 
![page](https://user-images.githubusercontent.com/3336728/63203524-8f1b3e00-c098-11e9-847c-cc323023fdfe.jpg)

And no data gets imported.
The same happens for **Export / Import (CSV)**

So basically the **Export** feature works fine, as the exported files (XLSX/CSV) are perfect, but the **Import** doesn't.

Tested on: https://github.com/nivocart/nivocart/commit/084b075c65cffad3742b1c4450368ec2aafedcbf
PHP 7.0.33,

**Comment by nivocart** [comment]
Manually deleting database entries is never a good idea. It is always better to Disable them instead.
Why? because IDs are incremental and once deleted they cannot be re-used.
This is why the option "No (Delete all old data before Import)" is there, to do just that by moving the entries to the next available IDs.

Also, we must always keep in mind that tables are inter-connected.
Changing IDs in "Product Options" will inevitably have consequences in the "Product" table.

When using CSV export/import you shouldn't ever change (or delete) IDs. It is clearly stated on the page. CSV Export/Import should only be used for minor corrections, never for complete imports, as it is not designed to do that.

All in all, XLSX export/import works fine if used properly but it cannot protect from breaking the site. It is not a "Backup" system and it does have some limits.

This is not a bug.
